### PR TITLE
Bugfix/SSFR 100 default board creation page unmatch mokup

### PR DESCRIPTION
**Pull request by Kozgor**
- Visual updates according to design

### CODE REVIEW

File: src/components/ColumnConfiguration/ColumnConfiguration.tsx [hunk 1]
```diff
@@ -1,78 +1,132 @@
...
+  const column = columns.find(column => column.columnId === columnId);
+  const [isColumnTitleInput, setIsColumnTitleInput] = useState<boolean>(false);
+  const [isColumnSubtitleTextarea, setIsColumnSubtitleTextarea] = useState<boolean>(false);
+  const inputTitleRef = useRef<HTMLDivElement>(null);
+  const textareaSubtitleRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    if (isColumnTitleInput) {
+      if (inputTitleRef.current) {
+        const inputElement = inputTitleRef.current?.querySelector('input');
```
Comment (LinnykS) [inline src/components/ColumnConfiguration/ColumnConfiguration.tsx:23]: I assume, the condition for inputTitleRef.current can be simplified to a ternary operator in this line.

File: src/components/ColumnConfiguration/ColumnConfiguration.tsx [hunk 2]
```diff
@@ -1,78 +1,132 @@
...
+
+        if (inputElement) {
+          inputElement.focus();
+        }
+      }
+    }
+
```
Comment (LinnykS) [inline src/components/ColumnConfiguration/ColumnConfiguration.tsx:30]: Also, I suggest breaking this logic into two blocks and putting it into corresponding methods.

File: src/components/ColumnConfiguration/ColumnConfiguration.tsx [hunk 3]
```diff
@@ -1,78 +1,132 @@
...
+        const textareaElement = textareaSubtitleRef.current?.querySelector('textarea');
+
+        if (textareaElement) {
+          textareaElement.focus();
+        }
+      }
+    }
+  }, [isColumnTitleInput, isColumnSubtitleTextarea]);
+
   const subtitleHandler = (event: ChangeEvent<HTMLTextAreaElement>) => {
```
Comment (LinnykS) [inline src/components/ColumnConfiguration/ColumnConfiguration.tsx:42]: Seems like subtitleHandler and titleHandler contain similar logic. I think some general method can be used just by passing some variables in it: `titlesHandler(event: ChangeEvent<HTMLInputElement>, columns)`

**Review by LinnykS — CHANGES_REQUESTED** [review]

File: src/components/ColumnConfiguration/ColumnConfiguration.tsx (hunk 3 above)
Comment (Kozgor) [reply src/components/ColumnConfiguration/ColumnConfiguration.tsx]: I desided to add one more argument to the handler function to define wich field was used on change.

**Review by LinnykS — APPROVED** [review]

**Pull request merged**